### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by cwinkelmann**

### CODE REVIEW

File: wildlifemapper/dataloader_coco.py [hunk 1]
```diff
@@ -31,7 +31,8 @@ def __init__(self, img_folder, ann_file, image_set, transforms, return_masks):
         self.mosaic_border = [-self.img_size // 2, -self.img_size // 2]
 
     def __getitem__(self, idx):
-        if self.mosaic == '_train':
+        if self.mosaic == '_train': # TODO why is there an underscore
```
Comment (cwinkelmann) [inline wildlifemapper/dataloader_coco.py:34]: The mosaic augmentation is never used

File: wildlifemapper/segment_anything/modeling/__init__.py [hunk 2]
```diff
@@ -6,7 +6,8 @@
 
 from .sam import Sam
 from .image_encoder import ImageEncoderViT
-from .mask_decoder_org import MaskDecoderOrg
+# FIXME, why is this code supposed to be here?
+# from .mask_decoder_org import MaskDecoderOrg
```
Comment (cwinkelmann) [inline wildlifemapper/segment_anything/modeling/__init__.py:10]: Since masks are not used the import is not needed but fails to, so the repo code could have never run in the current state

File: wildlifemapper/visualize_prediction.py [hunk 3]
```diff
@@ -66,15 +74,21 @@
...
+
+# TODO why is the train data loader used in the validation loop?
 #build dataloader
-dataset_train = build_dataset(image_set='train', args=args)
-dataset_val = build_dataset(image_set='val', args=args)
-data_loader_train = DataLoader(dataset_train, batch_size=args.batch_size, 
-                               collate_fn=utils.custom_collate, num_workers=args.num_workers)
+# dataset_train = build_dataset(image_set='train', args=args)
+
+# dataset_val = build_dataset(image_set='val', args=args)
```
Comment (cwinkelmann) [inline wildlifemapper/visualize_prediction.py:82]: the original build_data returns the same data for train and val

File: wildlifemapper/dataloader_coco.py [hunk 4]
```diff
@@ -275,6 +275,7 @@ def __call__(self, image, target):
         return image, target
 
 #TODO : Add data augmentation later, transforms.py file from DETR
+## no augmentations are ever applied to the train set aside from flipping
```
Comment (cwinkelmann) [inline wildlifemapper/dataloader_coco.py:278]: despite having two files containing augmentations, the augmentation_yolo.py and augmentation.py.

File: wildlifemapper/segment_anything/predictor.py [hunk 5]
```diff
@@ -7,7 +7,7 @@
 import numpy as np
 import torch
 
-from segment_anything.modeling import Sam
+from ..segment_anything.modeling import Sam
```
Comment (cwinkelmann) [inline wildlifemapper/segment_anything/predictor.py:10]: previous import didn't resolve
